docs/codeql/codeql-cli/creating-codeql-databases.rst

```diff
@@ -200,6 +201,10 @@ commands that you can specify for compiled languages.
 
      codeql database create csharp-database --language=csharp --command='dotnet build /p:UseSharedCompilation=false /t:rebuild'
 
+- Go project built using a custom build script::
```

Comment:
This might not be hugely clear because both mechanisms are used at once. Perhaps provide two examples, one with a custom script (and therefore tracing enabled), and one using the default build command and using the environment variable to enable tracing (cross-check @sauyon that I haven't mixed up the logic here)

Reply:
This makes sense. I've added two examples; one with the `CODEQL_EXTRACTOR_GO_BUILD_TRACING=on` environment variable and one with a custom script.